### PR TITLE
Addition to 'Convert to new account models' doc 

### PR DESCRIPTION
**Pull request by zuluecho9**
DOC 6865. An emergency addition to help deflect high ticket count.

### CODE REVIEW

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 1]
```diff
@@ -20,16 +20,55 @@ If your users are on our [original user model](#user-models), you may have the a
...
+If one of those is true, you may have access to a self-serve pricing migration in the UI. To use that UI: 
 
-Have more questions? You can ask in the [Explorers Hub](https://discuss.newrelic.com/). 
+1. To find the UI option: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown), click **Manage your plan** and then click **Plans and pricing** ([direct link](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJhY2NvdW50SWQiOjI0NDEyODYsIm5lcmRsZXRJZCI6ImNvbW1lcmNlLW5lcmRsZXRzLm5yLXNob3AtZW50cnkiLCJwYXRoIjoiL3BsYW5zLWFuZC1wcmljaW5nIiwibWlncmF0aW9uRWxpZ2libGUiOnRydWUsImFjY291bnROYW1lIjoiU3RyYWRvcyBMYWJzIn0=&platform[accountId]=)). If you don't see an **Unlock full platform access** UI, contact your account representative or ask a question in the [Explorers Hub](https://discuss.newrelic.com/).
```
Comment (barbnewrelic) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:30]: This line has a direct link embedded. Is that intentional? Is that a generic " link that anyone can go to, or should you remove the ()[] and put the period after "Plans and pricing"?

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 2]
```diff
@@ -20,16 +20,55 @@ If your users are on our [original user model](#user-models), you may have the a
...
+### Downgrading full users to basic users
+
+For [New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/), your count of full users is one factor in billing. 
+
+For organizations on our original pricing plan, by default all users are full users, because full user count isn't a billing factor for that plan. So before you switch to the new pricing, you'll want to ensure that your users have the user type you want. To learn more about determining whether a user should be basic or full, see [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type). 
+
+Ideally you should update your users before switching to the new pricing. But full user count is pro-rated, so if you update user type soon after switching to new pricing, billing impact will be minimal.
```
Comment (barbnewrelic) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:39]: pro-rated s/be prorated no dash

**Review by barbnewrelic — APPROVED** [review]
See one line with a question and one line with a typo. Then feel free to merge!

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:30]: it's supposed to work for everyone. is what they told me anyway.

**Pull request merged**